thesis-de/content-de.tex

```diff
@@ -1479,12 +1479,18 @@ \subsubsection{Vorbereitung der Evaluierung}
   hier unabhängig von kostenpflichtigen Lizenzen und Copyright Rechten zu sein.
   Ebenfalls ist eine Open Source Software nötig, da hier der Quelltext der
   Software angepasst und verändert werden darf.
-  \1 Bei der Weboberfläche sollte ebenfalls auf die Performance geachtet werden,
-  so sollen Graphen und Werte schnell ohne große Zeitdifferenzen aus dem 
-  Datenhaltungssystem abgerufen werden können. Ebenso, sollte der Aufbau des
-  Dashboards beim neuladen der Seite, oder aber auch ein Wert z.B. der CPU 
-  Auslastung ändert sich massiv direkt angezeigt werden können, ohne einen 
-  Zeitlichen unterschied merken zu können.
+  \1 Der wichtigste Aspekt, bei der Performance der Weboberfläche, ist 
+  darauf zu achten, das Werte die visualisiert werden aus dem Zwischengespeicher
+  verwendet werden, um die Auslastung auf dem Datenhaltunssystem so gering wie
```

Comment:
Und das ist nicht präzise genug. Wie kommen die Daten in den Zwischenspeicher? Wie lange bleiben die dort? Wann wird entschieden, bzw. woher weiß der Zwischenspeicher, dass seine Informationen nicht mehr aktuell sind?